teamscale-maven-plugin/src/main/java/com/teamscale/tia/maven/TiaMojoBase.java

```diff
@@ -156,6 +156,24 @@ public abstract class TiaMojoBase extends AbstractMojo {
 	@Parameter(defaultValue = "false")
 	public boolean skip;
 
+	/**
+	 * Run all tests independent of TIA information.
+	 */
+	@Parameter(defaultValue = "false")
+	public boolean runAllTests;
+
+	/**
+	 * Run only impacted tests, if not, they might be ordered using the impact information.
```

Comment:
That comment does not make much sense to me. AFAIU if this is true it will either use Teamscale's selection+prioritization(TIA) (when runAllTests=false) or at least use Teamscale's prioritization (when runAllTests=true). If false it will not ask Teamscale at all and therefore also not perform any ordering etc.

Reply:
Adjusted.